src/domobserver.js

```diff
@@ -121,7 +121,7 @@ export class DOMObserver {
   }
 
   flush() {
-    if (!this.view.docView || this.flushingSoon) return
+    if (!this.view.docView || this.flushingSoon || !this.observer) return
```

Comment:
I think it'd make more sense to put the check in the next statement—it's possible for there to be events on the queue (via the chardata event handler) even when there isn't a mutation observer, and selection changes are also flushed through here.

Reply:
Done.